packages/client/src/components/World/index.tsx

```diff
@@ -32,29 +32,27 @@ import { SocketWebRTCClientTransport } from '../../transports/SocketWebRTCClient
 
 const engineRendererCanvasId = 'engine-renderer-canvas'
 
-const getDefaulEngineInitializeOptions = (): InitializeOptions => {
-  return {
-    publicPath: location.origin,
-    networking: {
-      schema: {
-        transport: SocketWebRTCClientTransport
-      } as NetworkSchema
-    },
-    renderer: {
-      canvasId: engineRendererCanvasId
-    },
-    physics: {
-      simulationEnabled: false,
-      physxWorker: new Worker('/scripts/loadPhysXClassic.js')
-    },
-    systems: [
-      {
-        type: SystemUpdateType.Fixed,
-        system: GolfSystem,
-        after: GameManagerSystem
-      }
-    ]
-  }
+const defaulEngineInitializeOptions: InitializeOptions = {
```

Comment:
I removed this because most of the places some change in the initialization option is required, like in golf or in map. We are also setting default options in the `initializeEngine` as well. It is your call now.